src/pages/AdoptablePetsDetails.jsx

```diff
@@ -1,18 +1,19 @@
 import React from "react";
 import PetDetail from "../components/PetDetail";
 import PetSidebar from "../components/PetSidebar";
-import { PetItems } from "../files/PetsData";
+import usePets from "../hooks/usePets";
 
 const AdoptablePetsDetails = props => {
-  const { id } = props.match.params;
-  const selectedPet = PetItems.find(item => item.id === id);
+  const { animalId } = props.match.params;
+  const [{ petItems, isLoading }] = usePets(`/hub/pets/${animalId}`);
+
   const {
     imageUrl,
     imageUrlAltText,
     aboutMe,
     attributes,
     animalName
-  } = selectedPet;
+  } = petItems;
```

Comment:
i believe this would probably be better named `pet`

Reply:
added a comment above, about this, if we don't change anything, our only other option would be to do

```js
const [{ petItems: pet, isLoading }] = usePets(`/hub/pets/${animalId}`);
```